companies/b3fuerthde.json

```diff
@@ -0,0 +1,26 @@
+{
+    "slug": "b3fuerthde",
+    "relevant-countries": [
+        "de"
+    ],
+    "categories": [
+        "school"
+    ],
+    "name": "Martin-Segitz-Schule, Staatliche Berufsschule 3",
+    "address": "Ottostr. 22\n90762 Fürth\nDeutschand",
+    "phone": "+49 911 756650",
+    "fax": " +49 911 7566555",
+    "email": "bbock@b3-fuerth.de",
```

Comment:
Unfortunately, we cannot have personal data (the DPO's name in this case) in our database (see [here](https://github.com/datenanfragen/data#data-sources)). In this case, I'm afraid, this means we will have to fallback to the generic `sekretariat@` and not have a suggested transport medium.

Reply:
I've fixed it.